packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -673,12 +684,12 @@ export const RemoteServerFactory = {
         let _require
 
         extensions.forEach((extension) => {
-            logger.info(`Loading extension: ${extension}`)
+            logger.info(`Loading extension: ${extension.name}`)
 
             const setupServerFilePath = path.join(
                 options.buildDir,
                 'extensions',
-                extension,
+                extension.name,
```

Comment:
I wonder if we need to have better terminology as to what we referring to here as 'name'. Extension instances have methods for `getName` which isn't the same as this value. 

Originally the idea was to have those shortcodes or fully qualified package names or paths. The later is not a thing anymore and the short code is also something that we've decided to punt on for now. But if that was the case that it could be one of those 3 things, what is a good variable name? identifier? .. thoughts.

Reply:
I see, right. In this case, I'll rename `extension.name` to `extension.packageName`.